efficientdet/keras/postprocess.py

```diff
@@ -160,7 +162,7 @@ def nms(params, boxes: T, scores: T, classes: T, padded: bool) -> List[T]:
 
   # TF API's sigma is twice as the paper's value, so here we divide it by 2:
   # https://github.com/tensorflow/tensorflow/issues/40253.
-  nms_top_idx, nms_scores, nms_valid_lens = tf.raw_ops.NonMaxSuppressionV5(
```

Comment:
I guess we still need to maintain the nms_valid_llens since we do have some paddings? Otherwise, users may not know how many boxes are valid in the returned values.

Reply:
As scores are padded by 0, so users could filter it by score again.

Reply:
Maybe we could output a ragged tensor. It's interesting.